docs/02-admin/01-installation/01-bare_metal.md

```diff
@@ -265,16 +265,19 @@ realpath_cache_size = 4096K
 realpath_cache_ttl = 600
 ```
 
-Optionally also enable OPCache for improved performances with PHP (recommended for both fpm and cli ini files):
```

Comment:
maybe not use parentheses `()` after existing parentheses ;). A new line or note should make more sense. Or even as a comment in the ini file `;`